pkg/manager/member/pvc_resizer.go

```diff
@@ -466,15 +464,17 @@ func (p *pvcResizer) resizeVolumes(ctx *componentVolumeContext) error {
 					return err
 				}
 				if !volumeExpansionSupported {
-					klog.Warningf("Skip to resize PVC %q of TC %q: storage class %q does not support volume expansion",
-						*pvc.Spec.StorageClassName, pvcID, tcID)
+					klog.Warningf("Skip to resize PVC %q of %q: storage class %q does not support volume expansion",
+						*pvc.Spec.StorageClassName, pvcID, id)
 					continue
 				}
 			} else {
-				klog.V(4).Infof("Storage classes lister is unavailable, skip checking volume expansion support for PVC %q of TC %q with storage class %s. This may be caused by no relevant permissions",
-					pvcID, tcID, *pvc.Spec.StorageClassName)
+				klog.V(4).Infof("Storage classes lister is unavailable, skip checking volume expansion support for PVC %q of %q with storage class %s. This may be caused by no relevant permissions",
+					pvcID, id, *pvc.Spec.StorageClassName)
 			}
 
+			allResized = false
```

Comment:
Move to L491?

Reply:
It is not a check, so I think it's ok.
```go
			mergePatch, err := json.Marshal(map[string]interface{}{
				"spec": map[string]interface{}{
					"resources": corev1.ResourceRequirements{
						Requests: corev1.ResourceList{
							corev1.ResourceStorage: quantityInSpec,
						},
					},
				},
			})
```

Reply:
or L495

Reply:
arrive here, it means we should resize the PVC, and the error of `json.Marshal` can be regarded as failed to resize

Reply:
But actually, before L495, this will be no use because the code will return, and no chance to check this var.